source/commands.rst

```diff
@@ -77,15 +92,16 @@ Troubleshooting
     :widths: 8, 8, 25
 
     /wg report,[-p],"Writes a report file (placed at ``plugins/WorldGuard/report.txt``) detailing information about the server setup (list of plugins, their versions, world settings) as well as WorldGuard's configuration. This command is to easily provide information to others for support purposes. The report does not contain sensitive data. Use ``-p`` to also submit the report to a pastebin site and have a link generated that you can give to others."
-    /wg profile,[-p] [-t <name>] [<minutes>],"Starts profiling CPU usage of the current running server, and runs the profiler for the given duration (defaulting to 5 minutes if a duration is not specified). The profiler is based on `WarmRoast <https://github.com/sk89q/warmroast>`_.
+    /wg profile,[-p] [-i <milliseconds>] [-t <name>] [<minutes>],"Starts profiling CPU usage of the current running server, and runs the profiler for the given duration (defaulting to 5 minutes if a duration is not specified). The profiler is based on `WarmRoast <https://github.com/sk89q/warmroast>`_.
 
-    By default, results are only collected for the main thread where the world is 'ticked', but ``-t`` can be specified to filter by a different thread name (case in-sensitive). An asterisk (``*``) can be specified instead to profile all threads (i.e. ``-t *``).
+    By default, results are only collected for the main thread where the world is 'ticked', but ``-t`` can be specified to filter by a different thread name (case in-sensitive). An asterisk (``*``) can be specified instead to profile all threads (i.e. ``-t *``). By default, the profiler uses an intervall of 20 milliseconds, but ``-i`` can be used to specify a custom intervall in milliseconds between 1 and 100.
```

Comment:
I'm not *entirely* sure what the interval does exactly, and looking at the code I didn't really get it either. So I left this vague, assuming people who change this flag know what it does.

Reply:
Sampling interval, i.e. how often the stack is dumped.